activity.html

```diff
@@ -0,0 +1,237 @@
+    <a class="item" href="history.html">History</a>
+    <a class="item" href="http://openwisp.io/docs">Docs</a>
+    <a class="item" href="support.html">Support</a>
+    <a class="item" href="thecode.html">Code</a>
+    <a class="item active" href="activity.html">Activity</a>
+    <a class="item" href="about.html">People</a>
+  </div>
+  <div class="pusher">
+    <div class="ui grid" id="top-bar">
+      <div class="ui computer only column sixteen wide">
+        <div class="ui center aligned container secondary menu">
+          <h1 class="item logo">
+            <a href="./">OpenWISP</a>
+          </h1>
+          <div class="right menu">
+            <a class="item" href="index.html">Home</a>
+            <a class="item" href="whatis.html">Features</a>
+            <a class="item" href="history.html">History</a>
+            <a class="item" href="http://openwisp.io/docs">Docs</a>
+            <a class="item" href="support.html">Support</a>
+            <a class="item" href="thecode.html">Code</a>
+            <a class="item active" href="activity.html">Activity</a>
+            <a class="item" href="about.html">People</a>
+          </div>
+        </div>
+      </div>
+      <div class="ui mobile tablet only sixteen wide column">
+        <div class="ui menu">
+          <h1 class="item logo">
+            <a href="./">OpenWISP</a>
+          </h1>
+          <div class="right menu">
+            <div class="menu-open item">
+              <i class="sidebar icon"></i>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="ui container" id="main">
+      <div class="ui large feed feed-container"></div>
+    </div>
+    <div class="ui black inverted segment padding-vertical" id="footer">
+      <div class="ui container stackable two column grid">
+        <div class="column logo white">
+          <a href="./">OpenWISP</a>
+        </div>
+        <div class="column social">
+          <a href="https://twitter.com/openwisp" class="ui twitter button">
+            <i class="twitter icon"></i>
+            Twitter
+          </a>
+          <a href="https://facebook.com/openwisp" class="ui facebook button">
+            <i class="facebook icon"></i>
+            Facebook
+          </a>
+          <a href="https://www.linkedin.com/groups/4777261" class="ui linkedin button">
+            <i class="linkedin icon"></i>
+            Linked In
+          </a>
+          <a href="https://github.com/openwisp" class="ui black button">
+            <i class="github icon"></i> Github
+          </a>
+        </div>
+        <p>&copy; 2008-2018 OpenWISP and individual contributors.</p>
+      </div>
+    </div>
+  </div>
+  <script src="js/jquery-1.7.1.min.js"></script>
+  <script src="js/semantic.min.js"></script>
+  <script src="js/scripts.js"></script>
+  <script>
+    function createEvent(avatar, name, event, repo, header, content, date) {
+      if (!event)
```

Comment:
`Event` can be false because I am passing it to the function as a parameter, and it's the string representation of that event. For example, `event` could be "created a new issue in". It's there because unhandled events would be empty strings and I didn't want to create a new message in the feed for it.